index.bs

```diff
@@ -1466,8 +1494,14 @@ Since the {{MLOperand/[[builder]]}} object is bound by the {{MLGraphBuilder/cons
 
 If an operation supports only a subset of {{MLOperandDataType}}s, the <dfn>allowed data types</dfn> for each of the operation's input operands, including both positional arguments and options, are given as either an explicit list of {{MLOperandDataType}}s, or a constraint that the operand's [=MLOperand/dataType=] must be the <dfn lt="same type as">same as</dfn> the [=MLOperand/dataType=] of another input operand, or <dfn lt="any data type">any</dfn> to allow any {{MLOperandDataType}}.
 
+Implementations may support additional data types for operands. This can be queried for each operation using the {{MLContext/opSupportLimits()}} method on {{MLContext}} and inspecting the {{MLDataTypeLimits/dataTypes}} value of the corresponding member for the operation.
+
+Issue: Can conforming implementations support *fewer* data types for an operation than what is listed for their [=allowed data types=]? In other words, is the list of allowed types only a recommendation?
```

Comment:
As currently implemented in Chromium "allowed data types" is an upper bound, not a lower bound. If a platform supports more than what is defined in the spec the extra data types are not exposed.

There was discussion at TPAC 2024 about defining "required data types" for operators as well, which would be a required subset and be the only supported types unless the site explicitly opted into performing the feature detection using `opSupportLimits()` to check whether additional data types were supported.

Reply:
Okay. So possible phrasing is:

> Implementations may support fewer data types for operands than specified. This can be queried for each operation using the {{MLContext/opSupportLimits()}} method on {{MLContext}} and inspecting the {{MLDataTypeLimits/dataTypes}} value of the corresponding member for the operation.
> 
> Issue: Can we define the subset of data types that are required for each operator?

Thoughts? Anywhere else we need to sprinkle "may" and "should" ?

Reply:
That's enough "may" and "should" for now.

Reply:
Done in 7328146